plugins/action/include/plugins/action/action.h

```diff
@@ -138,7 +138,10 @@ class Action : public PluginBase {
     Result return_to_launch() const;
 
     /**
-     * @brief Send command to reposition the vehicle to a specific WGS84 global position
+     * @brief Send command to reposition the vehicle to a specific position.
```

Comment:
Slight improvement:

```
    /**
     * @brief Send command to move the vehicle to a specific global position.
     *
     * The latitude and longitude are given in degrees (WGS84 frame) and the altitude
     * in meters AMSL (above mean sea level).
     *
     * @param latitude_deg Latitude in degrees
     * @param longitude_deg Longitude in degrees
     * @param altitude_amsl_m Altitude AMSL in meters
     * @param yaw_deg Yaw angle in degrees
     *
     * @return Result of request.
     */
```

What is the frame for the yaw angle?

Reply:
Thanks! I'll add that, it's relative to North clockwise.